cni-plugin/integration/run.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+cd "${BASH_SOURCE[0]%/*}"
+
+# Run kubectl with the correct context.
+function k() {
+  if [ -n "${TEST_CTX:-}" ]; then
+    kubectl --context="$TEST_CTX" "$@"
+  else
+    kubectl "$@"
+  fi
+}
+
+function create_test_lab() {
+    echo '# Creating the test lab...'
+    k create ns cni-plugin-test
+    k create serviceaccount linkerd-cni
+    # TODO(stevej): how can we parameterize this manifest with `version` so we
+    # can enable a testing matrix?
+    k create -f manifests/linkerd-cni.yaml
+}
+
+function cleanup() {
+    echo '# Cleaning up...'
+    k delete -f manifests/linkerd-cni.yaml || echo "could not delete -f manifests/linkerd-cni.yaml"
+    k delete serviceaccount linkerd-cni || echo "could not delete serviceaccount linkerd-cni"
+    k delete ns cni-plugin-test || echo "could not delete namespace cni-plugin-test"
+}
+
+trap cleanup EXIT
+
+if k get ns/cni-plugin-test >/dev/null 2>&1 ; then
+  echo 'ns/cni-plugin-test already exists' >&2
+  exit 1
+fi
+
+create_test_lab
+
+# Wait for linkerd-cni daemonset to complete
+if ! k rollout status --timeout=30s daemonset/linkerd-cni -n linkerd-cni; then
+  echo "!! linkerd-cni didn't rollout properly, printing logs";
+  k describe ds linkerd-cni
+  k logs linkerd-cni -n linkerd-cni
```

Comment:
we don't need a printf || echo here, do we?

Reply:
Not strictly since if line 44 fails, line 45 would almost certainly fail but I think I'd like to err on having more information so I've put `|| echo` on these lines as well.